smartsim/entity/dbnode.py

```diff
@@ -176,7 +176,7 @@ def _parse_db_host(self, filepath: t.Optional[str] = None) -> str:
             try:
                 ip = self._parse_ips(filepath, 1)[0]
             # suppress error
-            except FileNotFoundError:
+            except (FileNotFoundError, IndexError):
```

Comment:
Since this was a bit of red herring, should we remove this handle for `IndexError`s just to revert to the original behavior?

Reply:
Done!